test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -4336,3 +4336,27 @@ Describe 'Invoke-WebRequest and Invoke-RestMethod support Cancellation through C
         RunWithCancellation -Command 'Invoke-RestMethod' -Uri $uri
     }
 }
+
+Describe "Web cmdlets Unix Sockets tests" -Tags "CI", "RequireAdminOnWindows" {
+    BeforeAll {
+        $unixSocket = Get-UnixSocketName
+        $WebListener = Start-UnixSocket $unixSocket
+    }
+
+    It "Execute Invoke-WebRequest with -UnixSocket" {
+        $uri = Get-UnixSocketUri
+        $result = Invoke-WebRequest $uri -UnixSocket $unixSocket
+        $result.StatusCode | Should -Be "200"
+        $result.Content | Should -Be "Hello World Unix Socket."
+    }
+
+    It "Execute Invoke-RestMethod with -UnixSocket" {
+        $uri = Get-UnixSocketUri
+        $result = Invoke-RestMethod  $uri -UnixSocket $unixSocket
+        $result | Should -Be "Hello World Unix Socket."
+    }
+
+    AfterAll {
+        Remove-Item -Path $unixSocket -Force -ErrorAction Ignore
```

Comment:
```suggestion
        $unixSocket | Should -Not -Exist
```

Reply:
Actually, we can probably remove the `AfterAll` section, as the path is deleted when the socket closes.